src/app/ui/icons/ordinal-icon.tsx

```diff
@@ -1,13 +1,14 @@
 import { styled } from 'leather-styles/jsx';
 
-import { SvgProps } from '@app/ui/ui-types';
+import { IconProps } from './icon';
 
-export function OrdinalIcon({ size = 'xl', ...props }: SvgProps) {
+// TODO: Will refactor with AvatarIcons
```

Comment:
Maybe add a task for this refactor?

Reply:
I can link the issue in the comment, but I am already assigned/working on it.